packages/examples/package.json

```diff
@@ -24,17 +24,17 @@
     "graphql": "^15.5.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-relay": "^17.0.0",
-    "relay-runtime": "^17.0.0"
+    "react-relay": "^18.0.0",
```

Comment:
Why not `^18.1.0`?

Reply:
No specific reason, this is internal examples (resolved to 18.1 in lockfile anyway) so there is not real difference for consumers